README.md

```diff
@@ -16,12 +16,12 @@ For testing and development the package brings a minimal backend deployment. Thi
 
 Testing requires that [nodejs (with npm)](https://nodejs.org) and [docker](https://docker.com) are installed on the system. 
 
-On the command line runn the following commands
+On the command line run the following commands
 
 ```sh
 npm ci 
 npm run all
-docker compose -f docker-compose-integration.yaml up --force-recreate --remove-orphans
+docker compose -f docker-compose-local.yaml up --force-recreate --remove-orphans
```

Comment:
integration is the correct file.

Reply:
Oh my bad in that case. 
Sorry, I thought it was simply the Live / Production file and so I changed it, since I can't run it.

I'll change it back with the other changes that you suggessted ^^ 

(Am I supposed to be able to run it / Am doing something wrong?) 

<pre>[+] Running 6/11
<font color="#C01C28"> ⠿ indexer Error                                                                                                                                                                                              1.4s</font>
<font color="#D0CFCC"> ⠼ dspacer Pulling                                                                                                                                                                                            1.4s</font>
<font color="#07889A">   ⠿ 8740c948ffd4 Already exists                                                                                                                                                                              0.0s</font>
<font color="#07889A">   ⠿ 69038a8b17e6 Already exists                                                                                                                                                                              0.0s</font>
<font color="#07889A">   ⠿ 96bad108ab96 Already exists                                                                                                                                                                              0.0s</font>
<font color="#07889A">   ⠿ 04705a785ae4 Already exists                                                                                                                                                                              0.0s</font>
<font color="#07889A">   ⠿ c3400bc300d4 Already exists                                                                                                                                                                              0.0s</font>
<font color="#07889A">   ⠿ 2092ff9d50bb Pull complete                                                                                                                                                                               0.4s</font>
<font color="#D0CFCC">   ⠇ 94ce1d42ef7e Downloading     [=============&gt;                                     ]  14.45MB/55.13MB                                                                                                      0.8s</font>
<font color="#D0CFCC">   ⠇ 5dc065baf5e9 Download complete                                                                                                                                                                           0.8s</font>
<font color="#D0CFCC">   ⠇ 4f4fb700ef54 Download complete                                                                                                                                                                           0.8s</font>
Error response from daemon: pull access denied for sdgidx, repository does not exist or may require &apos;docker login&apos;: denied: requested access to the resource is denied
</pre>

I am logged into docker hub, but I assume I need access to a specific docker image or perhaps I need to log into the ZHAW docker repo?